templates/people/index.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello PeopleController!{% endblock %}
+
+{% block body %}
+    <div id="root">Loading...</div>
+    <noscript>To use this app, enable JavaScript</noscript>
+    <div>
+        Versijuojamas JavaScript failas: <b id="versionedFile">...</b>
+    </div>
+    <label>Vardas</label>
```

Comment:
Namų darbui nėra būtina, bet realiems projektams praverstų:
```suggestion
    <label for="name">Vardas</label>
```

Tada paspaudus ant vardo pasižymėtų atitinkamas laukelis. Be to neregiams skaityklės pasiūlytų tinkamą laukelį

Plačiau:
 * https://www.w3schools.com/tags/att_label_for.asp